js/hoverzoom.js

```diff
@@ -449,13 +452,41 @@ var hoverZoom = {
                     video.src = imgDetails.url;
                     imgFullSize = $(video).appendTo(hz.hzImg);
                     video.addEventListener('error', imgFullSizeOnError);
-                    video.addEventListener('loadedmetadata', posImg);
+                    video.addEventListener('loadedmetadata', function(){
+                        posImg();
+                        track = this.addTextTrack("captions", "English", "en");
```

Comment:
Extract this into a function and put behind a new option (disabled by default). Changes like this shouldn't suddenly appear for 280k current users.